vnext/Microsoft.ReactNative/Fabric/Composition/CompositionViewComponentView.cpp

```diff
@@ -1499,6 +1499,9 @@ void CompositionViewComponentView::updateProps(
   if (oldViewProps.opacity != newViewProps.opacity) {
     m_visual.Opacity(newViewProps.opacity);
   }
+  if (oldViewProps.testId != newViewProps.testId) {
+    m_visual.Comment(newViewProps.testId == "" ? L"View" : winrt::to_hstring(newViewProps.testId));
```

Comment:
Not sure that we want to be adding the overhead of setting a comment on every View. - And as it is this wont set it to View unless the user sets it to something other than empty, then sets it to empty.

I think its fine to just keep them empty by default.